addons/hr_payroll/models/resource.py

```diff
@@ -16,6 +17,24 @@ class ResourceCalendar(models.Model):
         'resource.calendar.attendance', 'calendar_id', 'Employees working Time',
         domain=[('resource_id', '!=', False)])
 
+    @api.multi
+    def transfer_leaves_to(self, other_calendar, resource=None, from_date=None):
```

Comment:
rename argument `resource` as it is not obvious that you can pass multiple resources